### DISCUSSION TITLE
Unifi-Protect 2.0.0 - Error

### DISCUSSION THREAD

**Discussion by Wubarius** [briis/unifiprotect#478] [Q&A]
Hi,

first - thx for unifi-protect integration - it worked great until the update 2.0.0.
Unifi-Protect Release 2.0.0 was released on May 26th, 2022.
Please check and update Protect Integration.

Logger: homeassistant.config_entries
Source: custom_components/unifiprotect/data.py:93
Error setting up entry UDMPRO for unifiprotect
ValueError: 'package' is not a valid SmartDetectObjectType

Thanks....
Thomas

**Comment by briis** [comment]
Hi Thomas,
As this is now part of Home Assistant Core, you should log all issues there: https://github.com/home-assistant/core/issues

But this is solved as part of Home Assistant 2022.5.5, so please upgrade to that version.

**Comment by Wubarius** [comment]
Hi,

thx für info and link. Problem not fixxed on my side.
I am run anytime latest version's.

Will keep the point open in the HA gitlab. You can close here.

Thanks...
Thomas
